fedimint-core/src/encoding/threshold_crypto.rs

```diff
@@ -0,0 +1,121 @@
+use std::io::{Error, Read, Write};
```

Comment:
I take it you just copied this whole file over.

Reply:
Yes